kotsadm/pkg/gitops/gitops.go

```diff
@@ -43,6 +43,13 @@ type GitOpsConfig struct {
 	IsConnected bool   `json:"isConnected"`
 }
 
+type GlobalGitopsConfig struct {
```

Comment:
Would recommend using GitOps instead of Gitops to keep the pattern going (noticed this in some other places as well in the pr)